crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -2576,22 +2606,31 @@ fn has_cjs_export(p: &Program) -> bool {
                     .map_or(true, |import| !is_turbopack_helper_import(import))
             })
         }) {
-            return false;
+            return DetectedDynamicExportType::UsingModuleDeclarations;
         }
     }
 
     struct Visitor {
+        cjs: bool,
+        value: bool,
+        namespace: bool,
         found: bool,
     }
 
     impl Visit for Visitor {
         visit_obj_and_computed!();
 
         fn visit_ident(&mut self, i: &Ident) {
-            if &*i.sym == "module"
-                || &*i.sym == "exports"
-                || &*i.sym == "__turbopack_export_value__"
-            {
+            if &*i.sym == "module" || &*i.sym == "exports" {
```

Comment:
Does this only work if the ident is unresolved? e.g. will it false positive `const module = {}`?

Reply:
Yes that's broken in that case. It's only a difference between no exports and commonjs exports, and doesn't matter at runtime. It generates a bit less code for "no exports", but nevermind...

Reply:
Can you add a comment explaining this, so we know that this is broken but it doesn't actually matter?